### PR TITLE
(fix)O3-3662:Prefill the user scope roles when editing

### PR DESCRIPTION
**Pull request by makombe**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Screenshots

[Screencast from 27-07-2024 03:37:26 ALASIRI.webm](https://github.com/user-attachments/assets/6d4bde29-4e13-4e36-8349-c82842d7d32d)

## Related Issue

https://openmrs.atlassian.net/browse/O3-3662

### CODE REVIEW

File: src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx [hunk 1]
```diff
@@ -372,7 +396,7 @@ const AddStockUserRoleScope: React.FC<AddStockUserRoleScopeProps> = ({
               <Checkbox
                 onChange={onEnabledChanged}
                 checked={formModel?.enabled}
-                labelText={t(`Enabled ?`)}
+                labelText={t("enabled", "Enabled ?")}
```
Comment (hadijahkyampeire) [inline src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx:399]: I think you need to update the translations

**Review by hadijahkyampeire — APPROVED** [review]
Otherwise looks good, thanks @makombe

File: src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx (hunk 1 above)
Comment (makombe) [reply src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx:400]: @hadijahkyampeire the translations already exits, it seems the key was drop and that is what I have restored and it works fine.

Comment (hadijahkyampeire) [reply src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx:400]: okay then, let's merge it

**Pull request merged**